pkg/processor/rbac/role_test.go

```diff
@@ -14,6 +14,11 @@ kind: ClusterRole
 metadata:
   creationTimestamp: null
   name: my-operator-manager-role
+aggregationRule:
```

Comment:
Please add aggregationRule sample to [test data](https://github.com/arttor/helmify/blob/main/test_data/k8s-operator-kustomize.output) and regenerate  helmify examples with command:
```
cat test_data/k8s-operator-kustomize.output | go run ./cmd/helmify examples/operator

Reply:
Done